rtb/analisi_dei_requisiti_ver0.4.0.typ

```diff
@@ -0,0 +1,732 @@
+- *Trigger*: l'amministratore intende visualizzare la posizione dei mezzi noleggiati.
+- *Scenario principale*:
+  + L'amministratore è collegato e autenticato alla _dashboard_ del sistema.
+  + La _dashboard_ mette a disposizione una mappa interattiva con i mezzi attualmente a noleggio, la cui posizione viene indicata attraverso dei _#rifGlossario("marker")_.
+#v(20pt)
+#figure(
+  image("../assets/use_cases/UC9.svg"),
+  caption: [Diagramma del caso d'uso UC9],
+)
+
+#pagebreak()
+
+=== UC10 - Visualizzazione degli annunci pubblicitari generati lato amministratore
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore è autenticato al sistema.
+- *Postcondizioni*: l'amministratore visualizza gli annunci generati dalla LLM per ogni utente con un noleggio attivo all'interno della mappa.
+- *Trigger*: l'amministratore vuole prendere visione di quanto generato da parte della LLM per i clienti del noleggio e per i punti di interesse convenzionati.
+- *Scenario principale*:
+  + L'amministratore accede alla mappa dei mezzi noleggiati.
+  + L'amministratore, tramite un "_click_" sul _marker_ che indica la posizione di un mezzo, visualizza l'annuncio pubblicitario generato per l'utente che ha attualmente in uso il mezzo.
+#v(20pt)
+#figure(
+  image("../assets/use_cases/UC10.svg"),
+  caption: [Diagramma del caso d'uso UC10],
+)
+
+//TODO: valutare se raffinare i casi d'uso con dei sottocasi
+
+//UC Perruz
+
+/*
+L'attore sarebbe il sistema => no UC
+Domanda: il capitolato chiede di avvisare l'admin, si può immaginare tramite notifica, però l'attore della notifica è il sistema quindi non andrebbe creato il caso d'uso. Come si fa ad implementare?
+*/
+#pagebreak()
+=== UC11 - Visualizzazione notifica di un annuncio generato
+- *Attore principale*: Amministratore.
+- *Precondizioni*: l'amministratore è autenticato e ha accesso alla _dashboard_ del sistema.
+- *Postcondizioni*: l'amministratore riceve una notifica relativa ad un annuncio generato.
+- *Trigger*: il sistema genera e invia all'utente un annuncio personalizzato relativo al punto di interesse in cui si trova.
+- *Scenario principale*: 
+  + Un utente si avvicina a un punto di interesse e il sistema genera un annuncio personalizzato che viene inviato all'utente.
+  + Nello stesso istante, nella _dashboard_, compare una notifica _pop-up_ visibile all'amministratore, con i seguenti dettagli:
+    - Utente destinatario.
+    - Punto di interesse associato.
+    - Data e ora di generazione dell'annuncio.
+- *Inclusioni*: 
+  + Scomparsa automatica della notifica di un annuncio generato (#link(<uc12>)[UC12]).
+  + Chiusura della notifica di un annuncio generato (#link(<uc13>)[UC13]).
+  + Apertura dello storico degli annunci dalla notifica (#link(<uc14>)[UC14]).
+
+// vedi uc11, l'amministratore che sarebbe attore non interagisce nemmeno
```

Comment:
Discuterei questo UC con Sync Lab. Onestamente la notifica all'amministratore, con le supposizioni fatte da Sync Lab su Big Data e quant'altro, non sta in piedi. Ti inonda di notifiche (discorso che avevo già fatto a fine sprint scorso)